client/state/ui/selectors.js

```diff
@@ -134,3 +134,7 @@ export function hasSidebar( state ) {
 	}
 	return get( getSection( state ), 'secondary', true );
 }
+
+export function shouldHideMasterbar( state ) {
+	return get( state.ui, 'hideMasterbar', false );
+}
```

Comment:
I don't have strong feelings (actually I'm a bit conflicted), but note that [there's a guideline](https://github.com/Automattic/wp-calypso/blob/master/client/state/selectors/README.md#state-selectors):

> [`client/state/selectors`] contains all available state selectors. Each file includes a single default exported function which can be used as a helper in retrieving derived data from the global state tree.

Reply:
I'm always confused when there is an existing file containing all the selectors. Happy to move this new one to its own file, but I always have in mind that I would be confused to find selectors for an area split across one big file and a couple of individual ones.